packages/moonstone/Scrollable/Scrollable.js

```diff
@@ -316,6 +327,10 @@ class ScrollableBase extends Component {
 		this.childRef = React.createRef();
 		this.uiRef = React.createRef();
 
+		this.state = {
```

Comment:
We might want to use imperative APIs to only update the state at the button rather than on the scrollable since this will trigger re-renders for everything within the scroller which can be expensive.